test/lib/ansible_test/_internal/sanity/yamllint.py

```diff
@@ -68,6 +70,16 @@ def test(self, args, targets, python_version):
         :type python_version: str
         :rtype: TestResult
         """
+        ansible_util.check_pyyaml(args, python_version)
+        if not ansible_util.CHECK_YAML_VERSIONS[python_version]['cloader']:
```

Comment:
Instead of checking this dictionary directly, update `check_pyyaml` to return a bool based on the result.

Reply:
Fixed.  Although note that `check_pyyaml()` checks for multiple features of pyyaml so I had to return a slightly more verbose object rather than a simple boolean.